submissions/AndriyBerezovskiy/hooli-style-popup/css/style.css

```diff
@@ -0,0 +1,218 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    font-family: Verdana, sans-serif;
+    background-color: #fbfbfb;
+  }
+  
+  ul,
+  ol {
+    list-style: none;
+  }
+  
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  .df {
+    display: flex;
+  }
+  
+  /* -----header----- */
+  
+  .header {
+    background-color: #eaeaea;
+  }
+  
+  .header__menu {
+    padding: 20px;
+  }
+  
+  .header__list {
+    justify-content: flex-end;
+    align-items: center;
+    gap: 15px;
+  }
+  
+  .header__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  
+    height: 30px;
+    width: 70px;
+    border: 1px solid transparent;
```

Comment:
:fire:

Reply:
Again a smile and I again doubt whether it means good)) I hope it was as above good))